.betterer.results

```diff
@@ -6668,6 +6668,9 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
     ],
+    "public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilters.tsx:5381": [
```

Comment:
I would love to find a way around these...

Reply:
TODO in https://github.com/grafana/grafana/issues/58375, unless someone else can find a way that doesn't refactor a bunch of external files.

Reply:
use [eslint ignore comments](https://raintank-corp.slack.com/archives/CHKLEJ668/p1667489134866859) instead of a betterer exception so the debt is apparent where it is

Reply:
@joshhunt I added the eslint-ignore coments, but betterer isn't removing these rules when I run the update?